flask_project/campaign_manager/static/js/project.js

```diff
@@ -0,0 +1,34 @@
+function deleteRequest() {
+  const popup = document.getElementById("delete-pop");
+  const background = document.getElementById("delete-background");
+  if (popup.style.display == "none" && background.style.display == "none") {
+    popup.style.display = "inline-table";
```

Comment:
It looks like the screenshot @russbiggs posted was because of this line. Would setting `display: block` work?

Reply:
Thank you!! Yes, It'll work (with more styling)!